### PR TITLE
Spelling error/grammar error fixed

### PR DESCRIPTION
**Pull request by mgifford**

### CODE REVIEW

File: _guide/onboarding-staff.md [hunk 1]
```diff
@@ -117,14 +117,14 @@ Onboarding Quiz (Internal) - [Edit view](https://docs.google.com/forms/d/1O7VV0Y
...
+  * Always speak loudly and make sure to overemphasize with your lips
   * With deaf people, look at the interpreter, not the person who you are talking with
   * Ask the person if there is anything you can do to communicate with them more effectively
   * All of the above
 
-### Trello card - Join the #accessibility slack channel & report your score
+### Trello card - Join the #free-and-open-source Slack channel & report your score
 
-We want everyone at CivicActions to get an 80% or higher in this quiz. If you don't score high enough, feel free to engage with the Accessibility Team or retake the exam.
+We want everyone at CivicActions to get an 80% or higher in this quiz. If you don't score high enough, feel free to engage with the Free Culture Team.
```
Comment (dmundra) [inline _guide/onboarding-staff.md:127]: @mgifford how come we don't want them to engage in the `#accessibility` channel?

File: _guide/onboarding-staff.md [hunk 2]
```diff
@@ -117,14 +117,14 @@ Onboarding Quiz (Internal) - [Edit view](https://docs.google.com/forms/d/1O7VV0Y
...
+  * Always speak loudly and make sure to overemphasize with your lips
   * With deaf people, look at the interpreter, not the person who you are talking with
   * Ask the person if there is anything you can do to communicate with them more effectively
   * All of the above
 
-### Trello card - Join the #accessibility slack channel & report your score
+### Trello card - Join the #accessibility Slack channel & report your score
 
-We want everyone at CivicActions to get an 80% or higher in this quiz. If you don't score high enough, feel free to engage with the Accessibility Team or retake the exam.
+We want everyone at CivicActions to get an 80% or higher in this quiz. If you don't score high enough, feel free to engage with the Free Culture Team.
```
Comment (dmundra) [inline _guide/onboarding-staff.md:127]: @mgifford can you revert this change as well?

File: _guide/onboarding-staff.md (hunk 1 above)
Comment (dmundra) [reply _guide/onboarding-staff.md]: Got it! Thanks for fixing that. There is one more below it 'Accessibility Team or retake the exam' changed to 'Free Culture Team'.

File: _guide/onboarding-staff.md (hunk 2 above)
Comment (mgifford) [reply _guide/onboarding-staff.md:127]: You can't get in the way of free culture Daniel!

File: _guide/onboarding-staff.md (hunk 1 above)
Comment (mgifford) [reply _guide/onboarding-staff.md]: Oops..  Thanks!

**Review by dmundra — APPROVED** [review]
